src/components/button-group/button-group.tsx

```diff
@@ -0,0 +1,7 @@
+import { FC } from 'react'
+import { ButtonGroupProps } from './types'
+import { ElButtonGroup } from './styles'
+
```

Comment:
**question:** Is there a reason you opted to put `ButtonGroupProps` in `types.ts` instead of adjacent to the component here?

**suggestion:** Unless the type is shared with other units of code in the `/button-group` folder, it's preferable to co-locate it with the component's implementation. Helps increase cohesion in the code.

Reply:
I was following the documentation's guidelines by separating interfaces and types into `types.ts` for consistency across the codebase.

Yes, it is not a shared type, and I agree to move `ButtonGroupProps` to the component to increase cohesion.

Reply:
TIL! Sorry, I had missed that recommendation in the guidelines. Might be something worth raising in the Elements dev chat on Teams; as per my feedback here, I don't think it makes sense to _always_ separate them.